### ISSUE TITLE
Fuselage cannot have z offset in front of the wing/fuselage junction (towards the nose)

### ISSUE THREAD

**Issue by kccwing** [ImperialCollegeLondon/sharpy#312]
**Describe the bug**
as described - structural solver nonconvergence observed

**To Reproduce**
branch dev_fuselage_bug - simple_HALE/generate_hale_nose.py vs simple_HALE/generate_hale_nose_dip.py

**Expected behaviour**
convergence and time-stepping until completion

**Screenshots**
|==========|==========|==========|==========|==========|==========|==========|==========|==========|==========|
|   iter   |alpha[deg]|elev[deg] |  thrust  |    Fx    |    Fy    |    Fz    |    Mx    |    My    |    Mz    |
|==========|==========|==========|==========|==========|==========|==========|==========|==========|==========|
 Residual is:    3.3313240830999096E-006
STOP Static equations did not converge (17235)

**System Info (please complete the following information):**
 - OS: [e.g. CentOS7/MacOS 13.1] HPC
 - SHARPy Version - latest

**Additional context**
prime suspect - CRV in this particular direction? Will do a bit more digging later - just starting this to keep it logged

**Comment by kccwing** [comment]
[dev_fuselage_bug](/ImperialCollegeLondon/sharpy/tree/dev_fuselage_bug)

**Comment by ben-l-p** [comment]
Only just looking at this now after we spoke before Christmas. I was having issues with the trim solver which I didn't get around to addressing, where the derivatives seem to be zero when they should be - do these issues also occur for dynamic cases?

**Comment by kccwing** [comment]
by derivatives you mean the gradients for the trim inputs from the Newton loop?

**Comment by ben-l-p** [comment]
Yes I do - the incorrect gradient caused some of the states for the next iteration to be something unreasonable which caused the structural solver to not converge. Might be a different issue

**Comment by kccwing** [comment]
hmmm incorrect as in diff in output/diff in input is incorrect or the result of input+output/gradient is too far away from the solution point? Assuming the former, changing the relaxation factor does help (some sort of a "step size" parameter), but only if you fix this bug

                    self.input_history[self.i_iter][i_dim] = (self.input_history[self.i_iter][i_dim]*(1 - self.settings['relaxation_factor']) +
                                                              self.input_history[self.i_iter][i_dim**_-1_**]*self.settings['relaxation_factor'])
